### PR TITLE
Feature/revenue 128 create rating district

### PR DESCRIPTION
**Pull request by Highpriest26**
My work

### CODE REVIEW

File: Frontend/src/Components/Settings/PropertyType/PropertyTypeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      setIsLoading(true);
+
+      const response = await apiCall({
+        endpoint: "property-type/create",
+        method: "post",
+        data: form,
+      });
+
+      if (response.status === 201) {
+        alert('Property Type created successfully!');
```
Comment (Skillz-systems) [inline Frontend/src/Components/Settings/PropertyType/PropertyTypeModal.tsx:35]: replace alert with a snackbar or a toast

File: Frontend/src/Components/Settings/PropertyUse/PropertyuseModal.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      setIsLoading(true);
+
+      const response = await apiCall({
+        endpoint: "property-use/create",
+        method: "post",
+        data: form,
+      });
+
+      if (response.status === 201) {
+        alert('Property Type created successfully!');
```
Comment (Skillz-systems) [inline Frontend/src/Components/Settings/PropertyUse/PropertyuseModal.tsx:35]: change alert to toast

File: Frontend/src/Components/Settings/RatingDistrict/RatingDistrictModal.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    try {
+      setIsLoading(true);
+      const response = await apiCall({
+        endpoint: "rating-district/create",
+        method: "post",
+        data: form,
+      });
+
+      if (response.status === 201) {
+        alert('Rating District created successfully!');
```
Comment (Skillz-systems) [inline Frontend/src/Components/Settings/RatingDistrict/RatingDistrictModal.tsx:34]: change alert to toast

File: Frontend/src/Components/Settings/Street/StreetModal.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    try {
+      setIsLoading(true);
+      const response = await apiCall({
+        endpoint: "street/create",
+        method: "post",
+        data: form,
+      });
+
+      if (response.status === 201) {
+        alert('Street created successfully!');
```
Comment (Skillz-systems) [inline Frontend/src/Components/Settings/Street/StreetModal.tsx:36]: change alert to toast

**Review by Skillz-systems — CHANGES_REQUESTED** [review]

**Review by kingsonly — APPROVED** [review]

**Review by Skillz-systems — APPROVED** [review]

**Pull request closed without merging**